### PR TITLE
[QPID-8208] Fix Sybase support for the link-store

### PR DESCRIPTION
**Pull request by overmeulen**

### CODE REVIEW

**Comment by k-wall** [comment]
Olivier, thanks for the PR, but QPID-8208 appears to be a typo (it relates to LDAP auth provider error handling rather than this problem). Can you correct and we'll review?

**Comment by overmeulen** [comment]
Strange, the issue I created is nowhere to be found... I'll create a new one and link this PR to it

File: broker-core/src/test/java/org/apache/qpid/server/store/BrokerStoreUpgraderAndRecovererTest.java [hunk 1]
```diff
@@ -131,6 +132,7 @@ public void testUpgradeVirtualHostWithJDBCStoreAndBoneCPPool()
         context.put("qpid.jdbcstore.bigIntType", "mybigint");
         context.put("qpid.jdbcstore.varBinaryType", "myvarbinary");
         context.put("qpid.jdbcstore.blobType", "myblob");
+        context.put("qpid.jdbcstore.timestampType", "mytimestamp");
```
Comment (alex-rufous) [inline broker-core/src/test/java/org/apache/qpid/server/store/BrokerStoreUpgraderAndRecovererTest.java]: Please undo the change as context variable "qpid.jdbcstore.timestampType" did not exist in previous model versions

File: broker-core/src/test/java/org/apache/qpid/server/store/BrokerStoreUpgraderAndRecovererTest.java [hunk 2]
```diff
@@ -165,6 +167,7 @@ public void testUpgradeVirtualHostWithJDBCStoreAndDefaultPool()
         hostAttributes.put("jdbcBigIntType", "mybigint");
         hostAttributes.put("jdbcBlobType", "myblob");
         hostAttributes.put("jdbcVarbinaryType", "myvarbinary");
+        hostAttributes.put("jdbcTimestampType", "mytimestamp");
```
Comment (alex-rufous) [inline broker-core/src/test/java/org/apache/qpid/server/store/BrokerStoreUpgraderAndRecovererTest.java]: Please undo the change as context variable "qpid.jdbcstore.timestampType" did not exist in previous model versions

File: broker-core/src/test/java/org/apache/qpid/server/store/BrokerStoreUpgraderAndRecovererTest.java [hunk 3]
```diff
@@ -191,6 +194,7 @@ public void testUpgradeVirtualHostWithJDBCStoreAndDefaultPool()
         context.put("qpid.jdbcstore.bigIntType", "mybigint");
         context.put("qpid.jdbcstore.varBinaryType", "myvarbinary");
         context.put("qpid.jdbcstore.blobType", "myblob");
+        context.put("qpid.jdbcstore.timestampType", "mytimestamp");
```
Comment (alex-rufous) [inline broker-core/src/test/java/org/apache/qpid/server/store/BrokerStoreUpgraderAndRecovererTest.java]: Please undo the change as context variable "qpid.jdbcstore.timestampType" did not exist in previous model versions

File: broker-plugins/jdbc-store/src/main/java/org/apache/qpid/server/store/jdbc/JDBCDetails.java [hunk 4]
```diff
@@ -379,7 +403,8 @@ public boolean isOverridden()
                 return contextMap.containsKey(CONTEXT_JDBCSTORE_USEBYTESFORBLOB)
                         || contextMap.containsKey(CONTEXT_JDBCSTORE_BIGINTTYPE)
                         || contextMap.containsKey(CONTEXT_JDBCSTORE_VARBINARYTYPE)
-                        || contextMap.containsKey(CONTEXT_JDBCSTORE_BLOBTYPE);
+                        || contextMap.containsKey(CONTEXT_JDBCSTORE_BLOBTYPE)
```
Comment (alex-rufous) [inline broker-plugins/jdbc-store/src/main/java/org/apache/qpid/server/store/jdbc/JDBCDetails.java]: Please undo the change as context variable for timestamp type should not be used to indicate to use bytes methods for blob.

Comment (overmeulen) [reply broker-plugins/jdbc-store/src/main/java/org/apache/qpid/server/store/jdbc/JDBCDetails.java]: The method modified here is isOverridden() not isUseBytesMethodForBlob()
